package.json

```diff
@@ -9,9 +9,8 @@
     "build": "nx run-many --target=build",
     "test": "nx run-many --target=test",
     "e2e": "nx run-many --target=e2e",
-    "lint:spell": "cspell --no-progress '**/*.md'",
-    "lint:check": "npx eslint . ; npm run lint:spell",
-    "lint:fix": "npx eslint --fix .",
+    "lint:check": "npx eslint . && npx cspell '**/*.md' --no-progress --show-suggestions --show-context && npx prettier --check '**/*.{css,pcss,html,md,ts,tsx,js,mjs,json}'",
+    "lint:fix": "npx eslint --fix . && npx prettier --write '**/*.{css,pcss,html,md,ts,tsx,js,mjs,json}'",
     "pr:report": "node ./scripts/reports/pr-report.mjs",
```

Comment:
@louis-bompart I did what you were suggesting and also I added the whole .json . I couldnt figure out how to add the {package,nx,project}.json} in the regex so whatever. Lets just lint all jsons.